### ISSUE TITLE
Custom Actions can't add/update answers

### ISSUE THREAD

**Issue by JustMaier** [amwmedia/plop#117]
Not sure if this was an intentional change or not, but in the latest version of Plop, you can no longer add to or update the answers object inside of custom actions.

For Example:

```js
plop.setGenerator('demo', {
	prompts: [
		{ type: 'input', name: 'name', message: 'Name' }
	],
	actions: [
		function (a) {
			a.demo = a.name + 'Demo';
			return plop.renderString('Added Demo', a);
		},
		{ type: 'add', path: '{{pluralize name}}Controller.cs', templateFile: 'templates/demo.hbs' }
	]
});
```

After the custom action is completed, `demo` is undefined in following actions and in templates.

**Comment by amwmedia** [comment]
Sorry for the delay in replying here, I agree that this is a defect that should be fixed. I'm going to try to look at it soon, I have a pretty good idea of where the change needs to be made. Also see #118 for another possibly interesting feature that could be used to address this.

**Comment by amwmedia** [comment]
this is fixed in [v2.1.0](https://github.com/amwmedia/plop/releases/tag/v2.1.0), released this morning.

**Comment by JustMaier** [comment]
Awesome thanks @amwmedia!
